data/sql/updates/pending_db_world/haggle.sql

```diff
@@ -0,0 +1,124 @@
+--
+DELETE FROM `creature_text` WHERE `CreatureID` = 14041;
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES
+(14041, 0, 0, 'Spare some change?', 12, 0, 100, 1, 0, 0, 9088, 0, 'Haggle - Random Say'),
+(14041, 0, 1, 'I had it all!  I had it all and then I lost it.  Lost... all gone... like... my mind.  My mind is like... cheese.  I like cheese.', 12, 0, 100, 1, 0, 0, 9089, 0, 'Haggle - Random Say'),
+(14041, 0, 2, 'Forty-two... forty-two... forty-two what?  It could be anything!  Forty-two... hmmm...', 12, 0, 100, 1, 0, 0, 9098, 0, 'Haggle - Random Say'),
+(14041, 0, 3, 'Rat-kabobs!  Get your tasty Rat-kabobs here!  Get them while they last!', 12, 0, 100, 1, 0, 0, 9150, 0, 'Haggle - Random Say'),
+(14041, 0, 4, 'Rats everywhere.  Everywhere I see rats.  Always looking at me with their beedy little eyes.  I\'ll show them.  I\'ll show them all!', 12, 0, 100, 1, 0, 0, 9149, 0, 'Haggle - Random Say'),
+(14041, 0, 5, '%s yawns.', 16, 0, 100, 0, 0, 0, 9417, 0, 'Haggle - Emote 1'),
+(14041, 1, 0, '%s sifts through the trash.', 16, 0, 100, 69, 0, 0, 9141, 0, 'Haggle - Emote 2');
+
+DELETE FROM `creature` WHERE `guid` = 53788;
+INSERT INTO `creature` (`guid`, `id1`, `id2`, `id3`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `currentwaypoint`, `curhealth`, `curmana`, `MovementType`, `npcflag`, `unit_flags`, `dynamicflags`, `ScriptName`, `VerifiedBuild`) VALUES
+(53788, 14041, 0, 0, 369, 0, 0, 1, 1, 0, 38.99483, 24.375648, -4.2973485, 0.366166770458221435, 190, 0, 0, 484, 0, 0, 0, 0, 0, '', 45704);
+
+DELETE FROM `waypoints` WHERE `entry` = 14041;
+INSERT INTO `waypoints` (`entry`, `pointid`, `position_x`, `position_y`, `position_z`, `orientation`, `delay`, `point_comment`) VALUES
+(14041, 1, 33.34522, 31.454027, -4.2973475, NULL, 0, 'Haggle'),
```

Comment:
Waypoint IDs are entry*100 just like actionlists. A lot of existing ones don't follow that rule tho 😤

Reply:
Oof that is why I did the same